docs/ops/security-ssl.md

```diff
@@ -22,22 +22,22 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-This page provides instructions on how to enable SSL for the network communication between different flink components.
+This page provides instructions on how to enable SSL for the network communication between different Flink components.
 
 ## SSL Configuration
 
-SSL can be enabled for all network communication between flink components. SSL keystores and truststore has to be deployed on each flink node and configured (conf/flink-conf.yaml) using keys in the security.ssl.* namespace (Please see the [configuration page](config.html) for details). SSL can be selectively enabled/disabled for different transports using the following flags. These flags are only applicable when security.ssl.enabled is set to true.
+SSL can be enabled for all network communication between Flink components. SSL keystores and truststore has to be deployed on each Flink node and configured (conf/flink-conf.yaml) using keys in the `security.ssl.*` namespace (Please see the [configuration page](config.html) for details). SSL can be selectively enabled/disabled for different transports using the following flags. These flags are only applicable when `security.ssl.enabled` is set to true.
 
 * **taskmanager.data.ssl.enabled**: SSL flag for data communication between task managers
 * **blob.service.ssl.enabled**: SSL flag for blob service client/server communication
-* **akka.ssl.enabled**: SSL flag for the akka based control connection between the flink client, jobmanager and taskmanager 
-* **jobmanager.web.ssl.enabled**: Flag to enable https access to the jobmanager's web frontend
+* **akka.ssl.enabled**: SSL flag for akka based control connection between the Flink client, JobManager and TaskManager 
```

Comment:
we typically write `jobmanager` and `taskmanager`, please revert

Reply:
will revert back! I was reading other pages and seems there is some inconsistency on other pages as they used `JobManager` and `TaskManager` fromt